modules/data.land/R/write_ic.R

```diff
@@ -55,7 +55,8 @@ write_ic <- function(in.path, in.name, start_date, end_date,
   # read in registration xml for pool specific information
   register.xml <- system.file(paste0("register.", model$type, ".xml"), package = paste0("PEcAn.", model$type))
   if(file.exists(register.xml)){
-    register     <- PEcAn.data.atmosphere::read.register(register.xml, con = NULL)
+    register     <- XML::xmlToList(XML::xmlParse(register.xml))
+    PEcAn.logger::logger.debug(as.data.frame(register))
```

Comment:
I assume this is leftover from testing the change
```suggestion
```

Reply:
My bad. I forgot to remove these.